src/base/static/components/input-form/index.js

```diff
@@ -67,22 +67,25 @@ class InputForm extends Component {
       ) >= 0;
     this.attachments = [];
     this.geometryStyle = null;
-    let fields = OrderedMap();
-    selectedCategoryConfig.fields.forEach(field => {
-      fields = fields.set(
-        field.name,
-        Map()
-          .set(constants.FIELD_VALUE_KEY, "")
-          .set(constants.FIELD_RENDER_KEY, Math.random()),
```

Comment:
Note that I had to remove our use of `Math.random()` here because snapshot testing depends on deterministic rendering. An alternative is to mock the `Math.random()` function, but I figured that incrementing an integer here instead is simpler.

In the long run, we should definitely remove the need for a `FIELD_RENDER_KEY`. There is definitely a solution here, I suppose we just haven't had time to explore it.

Reply:
Yeah, I suppose a simple counter is more straightforward than a random number.